src/common/hostPlatform.ts

```diff
@@ -53,6 +55,11 @@ class WindowsHostPlatform implements IHostPlatform {
     public getPlatformId(): HostPlatformId {
         return HostPlatformId.WINDOWS;
     }
+
+    public getTargetPlatformCompatibility(targetPlatformName: string): boolean {
```

Comment:
I don't mind it as it is, but perhaps something like `supportsTargetPlatform(targetPlatformName: string)` so it reads as `if (hostPlatform.supportsTargetPlatform(platform)) { ...}`